---
permalink: /publications/
title: "Publications"
---

2021

{: style="color:gray; font-size: 70%"}
* [MINTIE: identifying novel structural and splice variants in transcriptomes using RNA-seq data](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02507-8), *Genome Biology*  
**Marek Cmero**, Breon Schmidt, Ian J. Majewski, Paul G. Ekert, Alicia Oshlack^, Nadia M. Davidson^  
[code](https://github.com/Oshlack/MINTIE) | [analysis code](https://github.com/Oshlack/MINTIE-paper-analysis) | [bioconda](https://bioconda.github.io/recipes/mintie/README.html)
* [Characterizing genetic intra-tumor heterogeneity across 2,658 human cancer genomes](https://www.sciencedirect.com/science/article/pii/S0092867421002944), *Cell*  
Stefan C. Dentro, Ignaty Leshchiner, Kerstin Haase, Maxime Tarabichi, Jeff Wintersinger, Amit G. Deshwar, Kaixian Yu, Yulia Rubanova, Geoff Macintyre, Jonas Demeulemeester, Ignacio Vázquez-García, Kortine Kleinheinz, Dimitri G. Livitz, Salem Malikic, Nilgun Donmez, Subhajit Sengupta, Pavana Anur, Clemency Jolly, **Marek Cmero**, Daniel Rosebrock, Steven E. Schumacher, Yu Fan, Matthew Fittall, Ruben M.Drews, Xiaotong Yao, Thomas B.K. Watkins, Juhee Lee, Matthias Schlesner, Hongtu Zhu, David J. Adams, Nicholas McGranahan, Charles Swanton, Gad Getz, Paul C. Boutros, Marcin Imielinski, Rameen Beroukhim, S. Cenk Sahinalp, Yuan Ji, Martin Peifer, Inigo Martincorena, Florian Markowetz, Ville Mustonen, Ke Yuan, Moritz Gerstung, Paul T. Spellman, Wenyi Wang, Quaid D. Morris, David C. Wedge, Peter Van Loo, on behalf of the PCAWG Evolution and Heterogeneity Working Group and the PCAWG Consortium  
[code](https://github.com/PCAWG-11/Heterogeneity)
* [Loss of SNAI in Prostate Cancer Correlates With Clinical Response to Androgen Deprivation Therapy](https://ascopubs.org/doi/full/10.1200/PO.20.00337), *JCO Precision Oncology*  
**Marek Cmero**^, Natalie J. Kurganovs^, Ryan Stuchbery, Patrick McCoy, Corrina Grima, Anne Ngyuen, Ken Chow, Stefano Mangiola, Geoff Macintyre, Nicholas Howard, Michael Kerger, Philip Dundee, Paul Ruljancich, David Clarke, Jeremy Grummet, Justin S. Peters, Anthony J. Costello, Sam Norden, Andrew Ryan, Phillip Parente, Christopher M. Hovens and Niall M. Corcoran  


2020

{: style="color:gray; font-size: 70%"}
* [MINTIE: identifying novel structural and splice variants in transcriptomes using RNA-seq data](https://www.biorxiv.org/content/10.1101/2020.06.03.131532v1.abstract), *bioRxiv*  
**Marek Cmero**, Breon Schmidt, Ian J. Majewski, Paul G. Ekert, Alicia Oshlack, Nadia M. Davidson  
[code](https://github.com/Oshlack/MINTIE) | [analysis code](https://github.com/Oshlack/MINTIE-paper-analysis)
* [Prostate cancer cell‐intrinsic interferon signaling regulates dormancy and metastatic outgrowth in bone](https://www.embopress.org/doi/full/10.15252/embr.202050162), *EMBO Reports*  
Katie L. Owen, Linden J. Gearing, Damien J. Zanker, Natasha K. Brockwell, Weng Hua Khoo, Daniel L. Roden, **Marek Cmero**, Stefano Mangiola, Matthew K. Hong, Alex J. Spurling, Michelle McDonald, Chia‐Ling Chan, Anupama Pasam, Ruth J. Lyons, Hendrika M. Duivenvoorden, Andrew Ryan, Lisa M. Butler, John M. Mariadason, Tri Giang Phan, Vanessa M. Hayes, Shahneen Sandhu, Alexander Swarbrick, Niall M. Corcoran, Paul J. Hertzog, Peter I. Croucher, Chris Hovens, Belinda S. Parker
* [Inferring structural variant cancer cell fraction](https://www.nature.com/articles/s41467-020-14351-8), *Nature Communications*  
**Marek Cmero**^, Ke Yuan^, Cheng Soon Ong, Jan Schröder, PCAWG Evolution and Heterogeneity Working Group, Niall M. Corcoran, Tony Papenfuss, Christopher M. Hovens, Florian Markowetz, Geoff Macintyre, PCAWG Consortium  
[code](https://github.com/mcmero/SVclone) | [analysis code](https://github.com/mcmero/SVclone_Rmarkdown) | [ccube code](https://github.com/mcmero/SVclone_Rmarkdown) | [simulation code](https://github.com/mcmero/sv_simu_pipe) | [bioconda](https://bioconda.github.io/recipes/svclone/README.html)
* [The evolutionary history of 2,658 cancers](https://www.nature.com/articles/s41586-019-1907-7), *Nature*  
Moritz Gerstung, Clemency Jolly, Ignaty Leshchiner, Stefan C. Dentro, Santiago Gonzalez, Daniel Rosebrock, Thomas J. Mitchell, Yulia Rubanova, Pavana Anur, Kaixian Yu, Maxime Tarabichi, Amit Deshwar, Jeff Wintersinger, Kortine Kleinheinz, Ignacio Vázquez-García, Kerstin Haase, Lara Jerman, Subhajit Sengupta, Geoff Macintyre, Salem Malikic, Nilgun Donmez, Dimitri G. Livitz, **Marek Cmero**, Jonas Demeulemeester, Steven Schumacher, Yu Fan, Xiaotong Yao, Juhee Lee, Matthias Schlesner, Paul C. Boutros, David D. Bowtell, Hongtu Zhu, Gad Getz, Marcin Imielinski, Rameen Beroukhim, S. Cenk Sahinalp, Yuan Ji, Martin Peifer, Florian Markowetz, Ville Mustonen, Ke Yuan, Wenyi Wang, Quaid D. Morris, PCAWG Evolution & Heterogeneity Working Group, Paul T. Spellman, David C. Wedge, Peter Van Loo, PCAWG Consortium  
[code](https://github.com/PCAWG-11/Evolution)


2019

{: style="color:gray; font-size: 70%"}
* [Using equivalence class counts for fast and accurate testing of differential transcript usage](https://f1000research.com/articles/8-265/v2), *F1000 Research*  
**Marek Cmero**, Nadia M. Davidson, Alicia Oshlack  
[pipeline code](https://github.com/Oshlack/ec-dtu-pipe) | [analysis code](https://github.com/Oshlack/ec-dtu-paper) | [vignette](https://github.com/Oshlack/ec-dtu-paper/wiki/Vignette)

2018

{: style="color:gray; font-size: 70%"}
* [Portraits of genetic intra-tumour heterogeneity and subclonal selection across cancer types](https://www.biorxiv.org/content/10.1101/312041v4.abstract), *bioRxiv*  
Stefan C. Dentro, Ignaty Leshchiner, Kerstin Haase, Maxime Tarabichi, Jeff Wintersinger, Amit G. Deshwar, Kaixian Yu, Yulia Rubanova, Geoff Macintyre, Ignacio Vázquez-García, Kortine Kleinheinz, Dimitri G. Livitz, Salem Malikic, Nilgun Donmez, Subhajit Sengupta, Jonas Demeulemeester, Pavana Anur, Clemency Jolly, **Marek Cmero**, Daniel Rosebrock, Steven Schumacher, Yu Fan, Matthew Fittall, Ruben M. Drews, Xiaotong Yao, Juhee Lee, Matthias Schlesner, Hongtu Zhu, David J. Adams, Gad Getz, Paul C. Boutros, Marcin Imielinski, Rameen Beroukhim, S. Cenk Sahinalp, Yuan Ji, Martin Peifer, Inigo Martincorena, Florian Markowetz, Ville Mustonen, Ke Yuan, Moritz Gerstung, Paul T. Spellman, Wenyi Wang, Quaid D. Morris, David C. Wedge, Peter Van Loo


2017

{: style="color:gray; font-size: 70%"}
* [Investigating the evolution of structural variation in cancer](https://minerva-access.unimelb.edu.au/handle/11343/215901), *PhD Thesis*  
**Marek Cmero**, supervised by Christopher Hovens, Geoff Macintyre, Niall Corcoran, Cheng Soon Ong, Tony Papenfuss
* [SVclone: inferring structural variant cancer cell fraction](https://www.biorxiv.org/content/10.1101/172486v1), *bioRxiv*  
**Marek Cmero**, Cheng Soon Ong, Ke Yuan, Jan Schröder, Kangbo Mo, PCAWG Evolution and Heterogeneity Working Group, Niall M. Corcoran, Anthony Troy Papenfuss, Christopher M. Hovens, Florian Markowetz, Geoff Macintyre


2016

{: style="color:gray; font-size: 70%"}
* [Comparing nodal versus bony metastatic spread using tumour phylogenies](https://www.nature.com/articles/srep33918), *Nature Scientific Reports*  
Stefano Mangiola, Matthew K. H. Hong, **Marek Cmero**, Natalie Kurganovs, Andrew Ryan, Anthony J. Costello, Niall M. Corcoran, Geoff Macintyre & Christopher M. Hovens  
[code](https://github.com/stemangiola/PC-lymph-node-mets-evolution)
* [Reduction in expression of the benign AR transcriptome is a hallmark of localised prostate cancer progression](https://www.oncotarget.com/article/8915/text/), *Oncotarget*   
Ryan Stuchbery, Geoff Macintyre, **Marek Cmero**, Laurence M. Harewood, Justin S. Peters, Anthony J. Costello, Christopher M. Hovens, Niall M. Corcoran
* [A urinary microRNA signature can predict the presence of bladder urothelial carcinoma in patients undergoing surveillance](https://www.nature.com/articles/bjc2015472), *British Journal of Cancer*  
Nikhil Sapre, Geoff Macintyre, Michael Clarkson, Haroon Naeem, **Marek Cmero**, Adam Kowalczyk, Paul D. Anderson, Anthony J. Costello, Niall M. Corcoran, Christopher M. Hovens

2015

{: style="color:gray; font-size: 70%"}
* [Drosha controls dendritic cell development by cleaving messenger RNAs encoding inhibitors of myelopoiesis](https://www.nature.com/articles/ni.3293), *Nature Immunology*  
Timothy M. Johanson, Ashleigh A. Keown, **Marek Cmero**, Janet H.C. Yeo, Amit Kumar, Andrew M. Lew, Yifan Zhan, Mark M.W. Chong
* [Tracking the origins and drivers of subclonal metastatic expansion in prostate cancer](https://www.nature.com/articles/ncomms7605), *Nature Communications*  
Matthew K.H. Hong, Geoff Macintyre, David C. Wedge, Peter Van Loo, Keval Patel, Sebastian Lunke, Ludmil B. Alexandrov, Clare Sloggett, **Marek Cmero**, Francesco Marass, Dana Tsui, Stefano Mangiola, Andrew Lonie, Haroon Naeem, Nikhil Sapre, Pramit M. Phal, Natalie Kurganovs, Xiaowen Chin, Michael Kerger, Anne Y. Warren, David Neal, Vincent Gnanapragasam, Nitzan Rosenfeld, John S. Pedersen, Andrew Ryan, Izhak Haviv, Anthony J. Costello, Niall M. Corcoran, Christopher M. Hovens


2014

{: style="color:gray; font-size: 70%"}
* [A microRNA expression atlas of mouse dendritic cell development](https://asi.onlinelibrary.wiley.com/doi/abs/10.1038/icb.2014.109), *Immunology & Cell Biology*  
Timothy M Johanson, **Marek Cmero**, James Wettenhall, Andrew M Lew, Yifan Zhan, Mark MW Chong
* [Socrates: identification of genomic rearrangements in tumour genomes by re-aligning soft clipped reads](https://academic.oup.com/bioinformatics/article/30/8/1064/257640), *Bioinformatics*  
Jan Schröder, Arthur Hsu, Samantha E. Boyle, Geoff Macintyre, **Marek Cmero**, Richard W. Tothill, Ricky W. Johnstone, Mark Shackleton, Anthony T. Papenfuss.  
[code](https://github.com/jibsch/Socrates)


{: style="color:gray; font-size: 70%"}
^ *Equal contributions*
